//package com.sincetimes.redwar.game.comnunicate.protocol;
message RseQueryRefuseReceiveWishItem
{
	optional int32 ret  = 1;
}